src/recommendations/implementation.md

```diff
@@ -0,0 +1,27 @@
+---
+group: product-recommendations
+title: Implementation Steps
+ee_only: True
+---
+
+To help you implement product recommendations on your storefront, use the following workflow:
+
+1. **Deploy data collection to production**
+   -  **Why** - The more data you collect the more accurate your recommendations will be. As soon as you start collecting data, you can pull that data into your non-production environment and experiment with different recommendation types on different pages.
+   -  **How** - [Install the Product Recommendations module]({{ page.baseurl }}/recommendations/install-configure.html).
+   -  **Notes** - Deploying data collection will not change your storefront's appearance or your shopper's experience. Only creating and deploying recommendation units will introduce new sections on your storefront. Do not create recommendation units until you customize your template.
+
+1. **Customize the template to match your style**
+   -  **Why** - Your storefront represents your brand so make sure you modify the product recommendations template to match the style of your site.
+   -  **How** - [Customize]({{ page.baseurl }}/recommendations/customize.html) the product recommendations template.
+   -  **Notes** - By customizing the template you can specify your stylesheet, overwrite where a recommendation unit will appear on a page, and so on.
+
+1. **Test recommendations on your non-production environment**
+   -  **Why** - It is always a best practice to test a new technology on your non-production environment before you deploy to production. Testing recommendations on your non-production environment allows you to play with different recommendation unit types, positioning, and pages. Because you already deployed the data collection JavaScript to production, you can now pull behavioral data from production so the recommendations you create on your non-production environment reflect the shopping behavior of actual customers.
```

Comment:
```suggestion
   -  **Why** - It is always a best practice to test a new technology on your non-production environment before you deploy to production. Testing recommendations on your non-production environment allows you to play with different recommendation unit types, positioning, and pages. If you already deployed the data collection in production, you can pull recommendations based on that behavioral data while testing in pre-production environment, so that recommendation results are based on the shopping behavior of actual customers.
```